app/server/static/pages/users.js

```diff
@@ -0,0 +1,14 @@
+import Vue from 'vue';
+import Buefy from 'buefy';
+import Users from '../components/users.vue';
+import 'buefy/dist/buefy.css';
```

Comment:
The header color is white only in users page.
![image](https://user-images.githubusercontent.com/6737785/64345960-39e5a300-d02c-11e9-81f4-d898947d7297.png)

Reply:
Resolved: 

![users](https://user-images.githubusercontent.com/8159064/67330876-0632df80-f4eb-11e9-9ee5-fc9b5b15a047.png)